terraform/modules/bb_alarms/main.tf

```diff
@@ -0,0 +1,457 @@
+  alarm_name          = "${var.app}-${var.env}-status_check_system"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+
+  evaluation_periods  = "${var.alarm_status_check_failed_system_eval_periods}"
+  metric_name         = "StatusCheckFailed_System"
+  namespace           = "AWS/EC2"
+  period              = "${var.alarm_status_check_failed_system_period}"
+  statistic           = "Sum"
+
+
+  dimensions {
+    AutoScalingGroupName = "${var.asg_name}"
+  }
+
+  alarm_description = "System status check has FAILED for ${var.asg_name} ASG in APP-ENV: ${var.app}-${var.env}"
+
+  threshold         = "${var.alarm_status_check_failed_system_threshold}"
+  unit              = "Count"
+  treat_missing_data = "notBreaching"
+
+  alarm_actions      = ["${var.cloudwatch_notification_arn}"]
+  ok_actions         = ["${var.cloudwatch_notification_arn}"]
+}
+
+
+
+resource "aws_cloudwatch_metric_alarm" "healthy_hosts" {
+  count               = "${var.alarm_elb_no_backend_enable}"
+  alarm_name          = "${var.load_balancer_name}-elb-no-backend"
+  comparison_operator = "LessThanThreshold"
+  evaluation_periods  = "${var.alarm_elb_no_backend_eval_periods}"
+  metric_name         = "HealthyHostCount"
+  namespace           = "AWS/ELB"
+  period              = "${var.alarm_elb_no_backend_period}"
+  statistic           = "Average"
+  threshold           = "${var.alarm_elb_no_backend_threshold}"
+
+  alarm_description   = "No backends available for ${var.load_balancer_name} in ${var.vpc_name} in APP-ENV: ${var.app}-${var.env}"
+
+  dimensions {
+    LoadBalancerName = "${var.load_balancer_name}"
+  }
+
+  # We should always have a measure of the number of healthy hosts - alert if not
+  treat_missing_data = "breaching"
+  alarm_actions      = ["${var.cloudwatch_notification_arn}"]
+  ok_actions         = ["${var.cloudwatch_notification_arn}"]
+}
+
+resource "aws_cloudwatch_metric_alarm" "high_latency" {
+  count               = "${var.alarm_elb_high_latency_enable}"
+  alarm_name          = "elb-${var.load_balancer_name}-high-latency"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  evaluation_periods  = "${var.alarm_elb_high_latency_eval_periods}"
+  metric_name         = "Latency"
+  namespace           = "AWS/ELB"
+  period              = "${var.alarm_elb_high_latency_period}"
+  statistic           = "Average"
+
+  dimensions {
+    LoadBalancerName = "${var.load_balancer_name}"
+  }
+
+  threshold         = "${var.alarm_elb_high_latency_threshold}"
+  unit              = "Seconds"
+  alarm_description = "High latency for ELB ${var.load_balancer_name} in ${var.vpc_name} in APP-ENV: ${var.app}-${var.env}"
+
+  # "Missing data" means that we haven't had any measure of latency - alert if we don't
+  treat_missing_data = "breaching"
+  alarm_actions      = ["${var.cloudwatch_notification_arn}"]
+  ok_actions         = ["${var.cloudwatch_notification_arn}"]
+}
+
+resource "aws_cloudwatch_metric_alarm" "spillover_count" {
```

Comment:
Another dupe of an alarm already in [elb_alarms](https://github.com/CMSgov/bluebutton-web-deployment/blob/master/terraform/modules/elb_alarms/main.tf) module. Can be removed.